custom/icds_reports/static/js/angular-services/date-helper.service.js

```diff
@@ -33,10 +49,17 @@ window.angular.module('icdsApp').factory('dateHelperService', ['$location', func
         var months = monthsCopy;
 
         if (selectedYear === new Date().getFullYear()) {
+            // this variable is used to decide if current month should be added to months list of the current year
+            var addCurrentMonth = !isBetweenFirstAndThirdDayOfCurrentMonth(new Date());
+            var maxMonthInCurrentYear = addCurrentMonth ? (new Date().getMonth() + 1) : (new Date().getMonth());
```

Comment:
i couldn't tell where this function was called - confirming that it's been adjusted so that it won't ever be passed January of the current month on the 1st or 2nd? Else I could imagine the year would be present but the list of months would be empty.

Reply:
I will check on this once again and get back to it

Reply:
I checked it once again, On January 1st and 2nd, current year will not be added to the list of years. `selectedYear`  argument of this function is checked for this condition before being passed